app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -656,6 +649,10 @@ class HomeActivityTest {
     ).check(matches(withText(stringToMatch)))
   }
 
+  private fun getProfileId(internalProfileId: Int): ProfileId {
```

Comment:
Prefer: ``createProfileId`` since this isn't specifically getting something, it's actually a factory helper since it's creating something.

Reply:
Done